nalgebra-lapack/src/cholesky.rs

```diff
@@ -161,6 +161,7 @@ where
         lapack_check!(info);
 
         // Copy lower triangle to upper triangle.
+        // Using unsafe to ensure the bounds i and j are always valid indices,
```

Comment:
This comment doesn't make sense. The unsafe operation *requires* that the indices are in bounds; it does nothing to ensure that is actually the case. The comment should explain how we know it's the case.